third-party/gmp/Makefile

```diff
@@ -72,16 +66,11 @@ cleanall: FORCE
 	rm -rf build
 
 clobber: FORCE
-	rm -rf build install $(GMP_SUBDIR)
+	rm -rf build install)
```

Comment:
yeah, caught in https://github.com/chapel-lang/chapel/pull/7015 -- should have had you do a pre-commit review :)